go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/nginxinc/nginx-gateway-fabric
 
-go 1.21.3
+go 1.22.0
```

Comment:
Remind me why we decided to pin this one to the patch version instead of just major.minor? I know v1.21 went past 1.21.3, so we never picked up those version bumps for this file.

Reply:
They changed how it works in `1.21`, we now need to specify the patch version.
It's also now a minimum version, we were already building with `1.21.6` and `1.22.0` since it came out.
And if one of our dependencies updates to `1.22` we won't be able to build.
>At go 1.21 or higher:
The go line declares a required minimum version of Go to use with this module.
The go line must be greater than or equal to the go line of all dependencies.

go.dev/doc/toolchain#version
go.dev/ref/mod#go-mod-file-go